logdb/logdb_bench_test.go

```diff
@@ -162,104 +86,61 @@ func BenchmarkFakeDB_NewestBlockID(t *testing.B) {
 	}
 }
 
-// BenchmarkFakeDB_WriteBlocks creates a temporary database, performs some write only benchmarks and then deletes it
+// BenchmarkFakeDB_WriteBlocks creates a temporary database, performs some write + commit benchmarks and then deletes the db
 func BenchmarkFakeDB_WriteBlocks(t *testing.B) {
-	dbPath, err := createTempDBPath()
-	defer os.Remove(dbPath)
-
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	db, err := logdb.New(dbPath)
-	if err != nil {
-		t.Fatal(err)
-	}
+	db, err := createTempDB()
+	require.NoError(t, err)
 	defer db.Close()
 
 	b := new(block.Builder).Build()
+	w := db.NewWriter()
 
 	t.ResetTimer()
 	for i := 0; i < t.N; i++ {
-		for i := 0; i < 1_000; i++ {
+		for i := 0; i < 10_000; i++ {
 			b = new(block.Builder).
 				ParentID(b.Header().ID()).
 				Transaction(newTx()).
-				Transaction(newTx()).
 				Build()
 			receipts := tx.Receipts{newReceipt(), newReceipt()}
-
-			w := db.NewWriter()
-			if err := w.Write(b, receipts); err != nil {
-				t.Fatal(err)
-			}
-
-			if err := w.Commit(); err != nil {
-				t.Fatal(err)
-			}
+			require.NoError(t, w.Write(b, receipts))
+			require.NoError(t, w.Commit())
```

Comment:
Commit should be outside of inner loop, then we have batches, which is the same situation in production.

Reply:
Added both instances.